### PR TITLE
Pbc/cms

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description

CMS PBC docs

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/cart-and-checkout/manage-in-the-back-office/reference-information-add-content-items-to-cms-pages-and-blocks.md [hunk 1]
```diff
@@ -1,26 +1,6 @@
...
-template: back-office-user-guide-template
-originalLink: https://documentation.spryker.com/2021080/docs/content-item-widgets-templates-reference-information
-originalArticleId: d155b7b4-e4bc-4ec9-9bbd-0af7558dac90
-redirect_from:
-  - /2021080/docs/content-item-widgets-templates-reference-information
-  - /2021080/docs/en/content-item-widgets-templates-reference-information
-  - /docs/content-item-widgets-templates-reference-information
-  - /docs/en/content-item-widgets-templates-reference-information
----
+Content item widget template defines how a content item looks on the Storefront. The following templates are available by default. If you need a different template, a developer can create it by following [HowTo:  Create a content item widget template](/docs/pbc/all/content-management-system/{{page.version}}/tutorials-and-howtos/howto-create-cms-templates.html#content-item-widget-template).
```
Comment (lenadoc) [inline _includes/pbc/all/cart-and-checkout/manage-in-the-back-office/reference-information-add-content-items-to-cms-pages-and-blocks.md:1]: Please add the frontmatter

File: _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-cms-glue-api.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ To start feature integration, overview and install the necessary features:
 | NAME     | VERSION | REQUIRED SUB-FEATURE     |
```
Comment (lenadoc) [inline _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-cms-glue-api.md:9]: same here for the frontmatter. Please also check the files below - some of them are missing the frontmatter as well

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: _includes/pbc/all/cart-and-checkout/manage-in-the-back-office/reference-information-add-content-items-to-cms-pages-and-blocks.md (hunk 1 above)
Comment (andriitserkovnyi) [reply _includes/pbc/all/cart-and-checkout/manage-in-the-back-office/reference-information-add-content-items-to-cms-pages-and-blocks.md:1]: This is an include :)

File: _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-cms-glue-api.md (hunk 2 above)
Comment (andriitserkovnyi) [reply _includes/pbc/all/install-features/202204.0/install-glue-api/install-the-cms-glue-api.md:9]: same here

**Review by jeremyfourna — DISMISSED** [review]
OK for me

**Review by lenadoc — APPROVED** [review]

**Review by lenadoc — APPROVED** [review]

**Pull request merged**